### ISSUE TITLE
Issue with permalinks

### ISSUE THREAD

**Issue by Vanp** [TwoBeers/boozurk#9]
Hi there! I want to thank you for this great theme, that I'm using for my new project, <a target="_blank" href="http://www.comunicosmos.com">Comunicosmos</a>.

But when I've tried to change the default permalink settings to anything else but "predetermined", the blog lost its structure (please check the attached image). So I've come back to "predetermined" permalinks and want to know if you could help me with this issue.
![comunicosmos](https://f.cloud.github.com/assets/3534080/145621/1aa44164-746b-11e2-80fb-165a45174e37.png)

Thank you!

Vanessa

**Comment by ilmaren** [comment]
Hi Vanessa,
usually permalink are related to host settings, not to theme.
Did you check if there's any message on permalinks page when you set them? (something like "You should update your .htaccess now.")

For more info on permalinks check http://codex.wordpress.org/Settings_Permalinks_Screen

**Comment by Vanp** [comment]
Thanks Ilmaren! Following your suggestion, I've checked permalinks to be sure .htaccess message didn't display. I tested other themes and everything was ok with non default permalinks, so I've removed Boozurk, re-uploaded it and started from the beginning, checking the results for every single change I was doing. Now everything is ok! I think I messed around too much with the code these days :-)

Thanks!

Vanessa
